usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -106,17 +108,24 @@ def get_agency_overview(self):
                 "abbreviation",
                 "toptier_code",
```

Comment:
Sort value `agency_code` is listed in the API contract and sortable fields but is a 500 error
Sort value `missing_tas_accounts_total` is listed in the contract but is not accepted by the API (now `missing_tas_accounts_count` in the code)

Reply:
Yeah, there was some confusion originally over what was supposed to be there, but I'm just gonna include both.